cgi-bin/DW/Widget/QuickUpdate.pm

```diff
@@ -18,59 +18,36 @@ package DW::Widget::QuickUpdate;
 
 use strict;
 use base qw/ LJ::Widget /;
+use DW::Template;
 
-sub need_res { qw( stc/widgets/quickupdate.css ) }
```

Comment:
This is the only place I see this CSS file included, so you might as well remove it completely.